modules/cli/src/main/scala/scala/cli/commands/scalafix/ScalafixOptions.scala

```diff
@@ -0,0 +1,66 @@
+package scala.cli.commands.scalafix
+
+import caseapp.*
+import coursier.core.Version
+
+import scala.build.EitherCps.{either, value}
+import scala.build.errors.BuildException
+import scala.build.internal.FetchExternalBinary
+import scala.build.options.BuildOptions
+import scala.cli.ScalaCli.fullRunnerName
+import scala.cli.commands.shared.{HasSharedOptions, HelpGroup, HelpMessages, SharedOptions}
+import scala.cli.commands.{Constants, tags}
+import scala.util.Properties
+
+// format: off
+@HelpMessage(ScalafixOptions.helpMessage, "", ScalafixOptions.detailedHelpMessage)
+final case class ScalafixOptions(
+  @Recurse
+    shared: SharedOptions = SharedOptions(),
+
+  @Group(HelpGroup.Format.toString)
+  @Tag(tags.implementation)
+  @HelpMessage("Custom path to the scalafix configuration file.")
+  @Tag(tags.inShortHelp)
+  scalafixConf: Option[String] = None,
+
+  @Group(HelpGroup.Format.toString)
+  @Tag(tags.implementation)
+  @HelpMessage("Pass an argument to scalafix.")
+  @Tag(tags.inShortHelp)
+  scalafixArg: List[String] = Nil,
+
+  @Group(HelpGroup.Format.toString)
+  @Tag(tags.implementation)
+  @HelpMessage("Pass rules directly to scalafix.")
+  @Tag(tags.inShortHelp)
+  rules: List[String] = Nil,
+
+  @Group(HelpGroup.Format.toString)
+  @Tag(tags.should)
+  @HelpMessage("Check if sources are well organized")
```

Comment:
This message looks very specific to some rules.
```suggestion
  @HelpMessage("Fail the invocation instead instead of applying fixes")
```